source/reference/minio-mc/mc-mb.rst

```diff
@@ -77,22 +78,22 @@ Parameters
       mc mb ~/mydata/mydir
 
 .. mc-cmd:: --ignore-existing, p
-   
+   :optional:
 
-   Directs :mc:`mc mb` to do nothing if the bucket or directory already
-   exists.
+   Directs :mc:`mc mb` to do nothing if the bucket or directory already exists.
 
 .. mc-cmd:: --region
-   
+   :optional:
 
-   The region in which to create the specified bucket. Has no effect if the
-   specified :mc-cmd:`~mc mb ALIAS` is a filesystem directory.
+   The region in which to create the specified bucket. 
+   Has no effect if the specified :mc-cmd:`~mc mb ALIAS` is a filesystem directory.
+
+   If not specified, default value is ``us-east-1``.
 
 .. mc-cmd:: --with-lock, l
-   
+   :optional:
 
-   Enables :ref:`object locking <minio-object-locking>` on the specified 
-   bucket.
+   Enables :ref:`object locking <minio-object-locking>` on the specified bucket.
```

Comment:
Should add that this implies `--with-versioning`.

Reply:
Done